Project.toml

```diff
@@ -5,9 +5,13 @@ repo = "https://github.com/lanl-ansi/Juniper.jl.git"
 version = "0.7.0"
 
 [deps]
+Cbc = "9961bab8-2fa3-5c5a-9d89-47fab24efd76"
 Distributed = "8ba89e20-285c-5b6f-9357-94700520ee1b"
+GLPK = "60bf3e95-4087-53dc-ae20-288a0d20c6a6"
+Ipopt = "b6b21f68-93f8-5de0-b562-5493be1d77c9"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
```

Comment:
Shouldn't these only be in `[extras]` to be not required when installing Juniper?

Reply:
I don't know what's happening, they are not on my branch but they are on Github